packages/contracts-bedrock/scripts/checks/check-interfaces.sh

```diff
@@ -67,6 +67,7 @@ EXCLUDE_CONTRACTS=(
     "IDelayedWETH"
     "IL2ToL2CrossDomainMessenger"
     "ICrossL2Inbox"
+    "ISystemConfigInterop"
```

Comment:
Not really, the issue is that there are 2 initializer functions in the interface (because it has to match the functions in the corresponding contract). And `ISystemConfigInterop.initializer.selector` is called in a script file, which cannot compile because it doesn't know which initializer function is intended

Reply:
OK, I see.